components/parser/kmp/xml/src/commonMain/kotlin/io/github/composegears/valkyrie/parser/kmp/xml/XmlToImageVectorParser.kt

```diff
@@ -0,0 +1,77 @@
+package io.github.composegears.valkyrie.parser.kmp.xml
+
+import io.github.composegears.valkyrie.ir.IrColor
+import io.github.composegears.valkyrie.ir.IrFill
+import io.github.composegears.valkyrie.ir.IrImageVector
+import io.github.composegears.valkyrie.ir.IrPathFillType
+import io.github.composegears.valkyrie.ir.IrStroke
+import io.github.composegears.valkyrie.ir.IrStrokeLineCap
+import io.github.composegears.valkyrie.ir.IrStrokeLineJoin
+import io.github.composegears.valkyrie.ir.IrVectorNode
+import io.github.composegears.valkyrie.parser.common.PathParser
+
+object XmlToImageVectorParser {
+    fun parse(text: String): Result<IrImageVector> {
```

Comment:
jvm parser throw exception in case any error

I propose to make the same behavior for now:

```kotlin
    @Throws
    fun parse(text: String): IrImageVector {
        return XmlDeserializer.deserialize(text).toIrImageVector()
    }
```